openedx/features/caliper_tracking/tests/tests.py

```diff
@@ -30,7 +29,13 @@ class CaliperTransformationTestCase(TestCase):
         return_value='honor',
         autospec=True
     )
-    def test_caliper_transformers(self, mock_function):
+    @mock.patch(
+        'openedx.features.caliper_tracking.utils.get_team_url_from_team_id',
+        return_value="http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/teams/" +
+                     "#teams/Topic1ID/check-3429fc5983a84c8c8366a4477b03d91c",
+        autospec=True
+    )
+    def test_caliper_transformers(self, mock_function, mock):
```

Comment:
Please use meaningful names such as `user_mock` and `team_mock` for the added params

Reply:
Done